src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -0,0 +1,134 @@
+package frc.robot.subsystems.gpm;
+
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.simulation.BatterySim;
+import edu.wpi.first.wpilibj.simulation.DutyCycleEncoderSim;
+import edu.wpi.first.wpilibj.simulation.EncoderSim;
+import edu.wpi.first.wpilibj.simulation.RoboRioSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.ArmConstants;
+
+public class Arm extends SubsystemBase {
+    
+    private Mechanism2d m_wristDisplay; 
+    private MechanismRoot2d m_pivot;
+    private MechanismLigament2d m_stationary;
+    private MechanismLigament2d m_moving;
+
+    /** the master motor */
+    private final TalonFX motor = new TalonFX(ArmConstants.MOTOR_ID);
+    /** the slave motors */
+    private final TalonFX[] slaves = new TalonFX[ArmConstants.SLAVE_IDS.length];
+    private final DutyCycleEncoder encoder = new DutyCycleEncoder(ArmConstants.ENCODER_ID);
```

Comment:
Are we adding a CANcoder? It would help make sure the angle was always accurate.

Reply:
Instead of a duty cycle encoder.

Reply:
were not using cancoder for the arm